release/ray_release/configs/global_config.py

```diff
@@ -4,6 +4,8 @@
 
 class GlobalConfig(TypedDict):
     byod_ray_ecr: str
+    byod_ray_project: str
```

Comment:
could you change it to maybe `ray_cr_repo` and `ray_ml_cr_repo`?

"project" is used as a suffix on github https://github.com/ray-project and docker hub https://hub.docker.com/u/rayproject , which is part of the organization name.

Reply:
totally